### PR TITLE
Use extpp and support arrow 0.12

### PR DESCRIPTION
**Pull request by mrkn**

### CODE REVIEW

File: ext/arrow-pycall/extconf.rb [hunk 1]
```diff
@@ -12,9 +12,14 @@
...
 
+require "extpp"
 require "mkmf-gnome2"
 
-unless required_pkg_config_package("arrow-glib")
+unless required_pkg_config_package("arrow-glib",
+                                   debian: "libarrow-glib-dev",
+                                   redhat: "arrow-glib-devel",
+                                   homebrew: "apache-arrow-glib",
+                                   msys2: "arrow")
```
Comment (kou) [inline ext/arrow-pycall/extconf.rb]: We don't need this change because red-arrow does this.

**Review by kou — CHANGES_REQUESTED** [review]
Could you add "extpp" to gemspec and add ">= 0.12.0" to required red-arrow version?

File: ext/arrow-pycall/extconf.rb (hunk 1 above)
Comment (mrkn) [reply ext/arrow-pycall/extconf.rb]: OK. I'll revert this..

**Comment by mrkn** [comment]
I'm working on updating Dockerfile.

**Comment by kou** [comment]
You can fix the Docker related failure by rebasing on master.

**Comment by mrkn** [comment]
@kou I want to use rubylang/ruby as the base image.

**Comment by kou** [comment]
OK.

**Comment by mrkn** [comment]
@kou Would you please have a look, again?

**Review by kou — APPROVED** [review]
+1

**Pull request merged**